front/src/applications/operationalStudies/components/Scenario/Timetable.tsx

```diff
@@ -302,7 +375,33 @@ export default function Timetable({
             data-testid="scenarios-filter"
           />
         </div>
+        <button
+          type="button"
+          className={cx('multiselect-toggle', multiselectOn ? 'on' : '')}
+          onClick={() => setMultiselectOn(!multiselectOn)}
+        >
+          <BiSelectMultiple />
+        </button>
```

Comment:
Maybe we should not show this button if there is no train
```suggestion
        { !isEmpty(trainList) && 
          (<button
            type="button"
            className={cx('multiselect-toggle', multiselectOn ? 'on' : '')}
            onClick={() => setMultiselectOn(!multiselectOn)}
          >
            <BiSelectMultiple />
          </button>
        )}
```